scripts/zones/Mamook/mobs/Hundredfaced_Hapool_Ja.lua

```diff
@@ -0,0 +1,39 @@
+-----------------------------------
+-- Area: Mamook
+--  Mob: Hundredfaced Hapool Ja
+-----------------------------------
+require("scripts/globals/spell_data")
+require("scripts/globals/titles")
+require("scripts/globals/status")
+local ID = require("scripts/zones/Mamook/IDs")
+mixins = {require("scripts/mixins/job_special")}
+-----------------------------------
+local entity = {}
+
+entity.onMobFight = function(mob, target)
+    mob:addListener("MAGIC_USE", "SPAWN_CLONES", function(mobArg, target, spell, action)
+    local spellId = spell:getID()
+        -- Utsusemi: Ichi (3 clones)
+        if spellId == xi.magic.spell.UTSUSEMI_ICHI then
+            for i = ID.mob.HUNDREDFACED_HAPOOL_JA + 1, ID.mob.HUNDREDFACED_HAPOOL_JA + 3 do
```

Comment:
You don't need to use this id here, you can get it with mob:getID and then remove the new entry from the zone lua. Then it'll be shift proof

Reply:
```lua
local hundredfaced_hapool_ja = mob:getID()
for i = hundredfaced_hapool_ja + 1, hundredfaced_hapool_ja + 3 do
```
so that its only called once while being shift-proof per Zach's suggestion

Reply:
Thanks for all the feedback. The only other issue I can't seem to move past, is making the clones update enmity to the target. (Seems like target is being lost on the new mob spawn) But I'll push a commit a bit later with the suggestions proposed and we can go from there.

Reply:
we'd need to get target from the main NM rather than relying on input the function arg, because in the context of this cast the target is itself (its where the spell is being cast onto)

Reply:
I did some similar "target-finding" here: <https://github.com/LandSandBoat/server/blob/base/modules/custom/lua/mazurka_generates_enmity.lua#L24-L29>

Reply:
Teo fed you a very quick example here, but keep an eye on styling: <https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md#lua-misc--naming>